src/components/SideBar.js

```diff
@@ -0,0 +1,94 @@
+import { Heading, Text, Flex, Divider, Avatar, IconButton } from '@chakra-ui/react';
+
+import { useState } from 'react';
+
+import { FiMenu, FiBriefcase, FiUser } from 'react-icons/fi';
+import { FaScribd } from 'react-icons/fa';
+import { BsPatchQuestion } from 'react-icons/bs';
+import { RiTeamLine } from 'react-icons/ri';
+import NavItem from './NavItems';
+
+const SideBar = () => {
+  const [navSize, changeNaveSize] = useState('small');
+  return (
+    <Flex
+      pos="sticky"
+      h="90vh"
+      boxShadow="0 4px 12px 0 rgba(0,0,0,0.05)"
+      w={navSize == 'small' ? '75px' : '200px'}
+      flexDir="column"
+      justifyContent="space-between"
+      bgGradient="linear(to-t, #97FFCF, #38C6BD)"
+    >
+      <Flex
+        p="5%"
+        flexDir="column"
+        alignItems={navSize == 'small' ? 'center' : 'flex-start'}
+        as="nav"
+      >
+        <IconButton
+          color="white"
+          background="none"
+          _hover={{ textDecor: 'none', backgroundColor: '#97FFCF' }}
+          mt={5}
+          _focus="outline-none"
+          icon={<FiMenu />}
+          onClick={() => {
+            if (navSize === 'small') changeNaveSize('large');
+            else changeNaveSize('small');
+          }}
+        />
+        <NavItem
+          navSize={navSize}
+          icon={FaScribd}
+          title="Pending for approval"
+          description="Description for Pending for approval item"
+        />
+        <NavItem
+          navSize={navSize}
+          icon={RiTeamLine}
+          title="Team"
+          description="Description for Team item"
+        />
+        <NavItem
+          navSize={navSize}
+          icon={BsPatchQuestion}
+          title="Questons"
+          description="Description for Questions item"
+        />
+        <NavItem
+          navSize={navSize}
+          icon={FiBriefcase}
+          title="Company Info"
+          description="Description for Company info item"
+        />
+        <NavItem
+          navSize={navSize}
+          icon={FiUser}
+          title="My Profile"
+          description="Description for My profile item"
+        />
```

Comment:
Usually we create an variable that holds the menu items config, and render it like this:
```suggestion
const navItems = [{
   title: 'Pending for approval',
   description: 'Description for Pending for approval item',
   icon: FaScribd,
}, {
    ....
}];

{navItems.map({ title, description, icon }) => (
        <NavItem
          navSize={navSize}
          icon={icon}
          title={title}
          description={description}
        />
))}
```